docs/_templates/index.html

```diff
@@ -217,21 +217,128 @@ <h2 class="featurette-heading">One log, many analyses. <span class="text-muted">
                         in Memray - there are many more to choose from, allowing
                         you to view your data in the way that makes the most
                         sense for your specific needs.
-                    </p>.
+                    </p>
                 </div>
                 <div class="col-md-5">
                     <img class="featurette-image img-fluid mx-auto reveal_l"
                         src="{{ pathto('_static/images/stats_example.png', 1) }}" alt="Many reporters">
                 </div>
             </div>
 
-            <hr class="featurette-divider">
-
+            <div class="row featurette">
+                <h2 class="featurette-heading">Memray in the media</h2>
+                <div class="row">
+                    <br>
+                    <div class="card-columns">
+                        <a href="https://news.ycombinator.com/item?id=31102089" class="card">
+                            <div class="card-body">
+                              <h5 class="card-title">Memray: a memory profiler for Python</h5>
+                              <h6 class="card-subtitle mb-2 text-muted">Hacker News</h6>
+                            </div>
+                          </a>
+                        <a href="https://news.ycombinator.com/item?id=31102089" class="card">
+                            <div class="card-body">
+                                <h5 class="card-title">Memray: a memory profiler for Python</h5>
+                                <h6 class="card-subtitle mb-2 text-muted">Hacker News</h6>
+                            </div>
+                        </a>
+                        <a href="https://pythonbytes.fm/episodes/show/282/dont-embarrass-me-in-front-of-the-wizards" class="card">
+                                <div class="card-body">
+                                <h5 class="card-title">#282: Don't Embarrass Me in Front of The Wizards</h5>
+                                <h6 class="card-subtitle mb-2 text-muted">PythonBytes (podcast) - 5/3/22</h6>
```

Comment:
From an internationalization point of view, I'd strongly prefer ISO dates ("2022-05-03") or dates with the months spelled out ("May 3, 2022") over dates in the US mm/dd/yy format. Many of the user's we're targeting with this advertising will find the US format confusing and distracting.